README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/llipio/conjugator.svg?branch=master)](https://travis-ci.org/llipio/conjugator)
```

Comment:
I added this badge based on the assumption which you have travice account. I just changed reference from mine to `llipio`